File: assignments/src/main/java/membermanagement3/Member.java
package membermanagement3;

public interface Member {
    String getName();
    String getEmail();
    String getPhone();
    String getGrade();
    String getBenefit();

    public void update(String name,  String email, String phone);

    default void printInfo() {
        System.out.println("[" + getGrade() + "] " + getName() + " / " + getEmail() + " / " + getPhone() + " (혜택: " + getBenefit() + ")");
    }
}

File: practice/kotlin/src/main/kotlin/part2/member/Member.kt
package part2.member

// 회원 한 명의 정보

class Member(var name: String, var email: String, var phone: String) {
    // 아직 회원이 들어오지 않은 빈 칸을 만들 때는 부 생성자
    constructor() : this("", "", "")

    override fun toString(): String {
        return "[이름]: $name, [이메일]: $email, [연락처]: $phone"
    }
}